<ui version="4.0" >
 <author>Tarek Saidi</author>
 <class>EditGroupDialog</class>
 <widget class="QDialog" name="EditGroupDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>120</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>350</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Group Properties</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="1" column="0" >
      <widget class="QLabel" name="Label2" >
       <property name="text" >
        <string>Icon:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="Label1" >
       <property name="text" >
        <string>Title:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QComboBox" name="ComboIconPicker" />
     </item>
     <item row="1" column="3" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>172</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1" colspan="3" >
      <widget class="QLineEdit" name="EditTitle" />
     </item>
     <item row="1" column="2" >
      <widget class="QToolButton" name="Button_Icon" >
       <property name="text" >
        <string>></string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line1" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ButtonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>EditTitle</tabstop>
  <tabstop>ComboIconPicker</tabstop>
  <tabstop>Button_Icon</tabstop>
  <tabstop>ButtonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
